### PR TITLE
Add HJT module

### PR DESCRIPTION
**Pull request by urielsalis**

### CODE REVIEW

File: modules/hjt/hjt.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, cmd string, args []string) {
+	if len(args) == 0 {
+		return
+	}
+	pasteLink := args[0]
+
+	content, err := readFromUrlAsLowercase(pasteLink)
+	if err != nil {
+		_, err = ds.ChannelMessageSend(mc.ID, "Invalid URL")
```
Comment (LordRalex) [inline modules/hjt/hjt.go]: This is the wrong ID. It is ChannelID

File: modules/hjt/hjt.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		logger.Debug().Printf("Invalid url: %s\n%s", pasteLink, err)
+		return
+	}
+
+	db, err := database.Get()
+	if err != nil {
+		_, err = ds.ChannelMessageSend(mc.ID, "Failed to connect to database")
```
Comment (LordRalex) [inline modules/hjt/hjt.go]: This is the wrong ID. It is ChannelID

File: modules/hjt/hjt.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, cmd string, args []string) {
+	if len(args) == 0 {
+		return
+	}
+	pasteLink := args[0]
+
+	content, err := readFromUrlAsLowercase(pasteLink)
+	if err != nil {
+		_, err = ds.ChannelMessageSend(mc.ID, "Invalid URL")
+		logger.Debug().Printf("Invalid url: %s\n%s", pasteLink, err)
```
Comment (LordRalex) [inline modules/hjt/hjt.go]: We don't need this logging anymore

File: modules/hjt/hjt.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	var values []string
+	err = db.Table("hjt").Where("? LIKE CONCAT('%', LOWER(name) ,'%')", content).Pluck("value", &values).Error
+
+	if err != nil {
+		logger.Err().Printf("Failed to pull data from database\n%s", err)
+		return
+	}
+
+	if len(values) == 0 {
+		_, err = ds.ChannelMessageSend(mc.ID, "No matches found")
```
Comment (LordRalex) [inline modules/hjt/hjt.go]: This is the wrong ID. It is ChannelID

File: modules/hjt/hjt.go [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+		_, err = ds.ChannelMessageSend(mc.ID, "Failed to connect to database")
+		logger.Err().Printf("Failed to connect to database\n%s", err)
+		return
+	}
+
+	var values []string
+	err = db.Table("hjt").Where("? LIKE CONCAT('%', LOWER(name) ,'%')", content).Pluck("value", &values).Error
+
+	if err != nil {
+		logger.Err().Printf("Failed to pull data from database\n%s", err)
```
Comment (LordRalex) [inline modules/hjt/hjt.go:44]: Should tell user as well

File: modules/hjt/hjt.go [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+		return
+	}
+
+	if len(values) == 0 {
+		_, err = ds.ChannelMessageSend(mc.ID, "No matches found")
+	} else {
+		_, err = ds.ChannelMessageSend(mc.ID, strings.Join(values, ", "))
```
Comment (LordRalex) [inline modules/hjt/hjt.go]: This is the wrong ID. It is ChannelID

**Review by LordRalex — CHANGES_REQUESTED** [review]

**Comment by LordRalex** [comment]
You also need to add this to https://github.com/LordRalex/absol/blob/master/core/module.go#L17
otherwise, we will not load this.

**Comment by urielsalis** [comment]
Done

**Pull request merged**